README.Rmd

```diff
@@ -29,80 +29,90 @@ knitr::opts_chunk$set(
 [![DOI](http://joss.theoj.org/papers/10.21105/joss.00971/status.svg)](https://doi.org/10.21105/joss.00971)
   <!-- badges: end -->
 
+`{piggyback}` provides an R interface for storing files as GitHub release assets, 
```

Comment:
@cboettig most of the prose changes are here in the README.md, it may be helpful to read the web version as well https://github.com/ropensci/piggyback/blob/d80b456929274ba27649b3def927eb5baf24eefa/README.md

Reply:
I believe the reason the authentication link doesn't work is because I renamed intro.Rmd to piggyback.Rmd (to take advantage of pkgdown Getting Started) so the existing pkgdown won't rebuild until this gets merged

Reply:
I think I'm going to merge this and wait for the ropensci pkgdown actions to catch and rebuild this, if this URL doesn't work then I'll come back and fix it

Reply:
Happy to add another sentence on auth to go along with it though.